x-pack/plugins/lens/public/indexpattern_plugin/to_expression.ts

```diff
@@ -25,16 +23,20 @@ export function toExpression(state: IndexPatternPrivateState) {
     return operationDefinition.toEsAggsConfig(column, columnId);
   }
 
-  if (sortedColumns.length) {
-    const aggs = sortedColumns.map((col, index) => {
-      return getEsAggsConfig(col, state.columnOrder[index]);
+  const columnEntries = state.columnOrder.map(colId => [colId, state.columns[colId]]);
```

Comment:
nit: If you specify the type of the entries as tuple here, you don't need the type casts in the lines below:
`const columnEntries = state.columnOrder.map(colId => [colId, state.columns[colId]] as [string, IndexPatternColumn]);` - one type cast less :)